sc/social/like/subscribers.py

```diff
@@ -107,3 +114,50 @@ def assign_canonical_url(obj, event):
             'Canonical domain not set in Social Media configlet; '
             "Facebook's Open Graph canonical URL (og:orl) will not be available"
         )
+
+
+def social_content_check(obj, event):
+    """Check if content is valid if enabled in sc.social.content, in action when edit."""
+
+    request = getRequest()
+
+    try:
+        view = getMultiAdapter((obj, request), name='social_likes_view')
+    except ComponentLookupError:
+        logger.info('not instaled.')
```

Comment:
I would change the message to 'social_likes_view not installed'. Having a string 'not installed' without other information isn't too much useful...

Reply:
that's not needed at all; just do nothing and return.